packages/orion/src/TagsInput/TagsInput.stories.js

```diff
@@ -62,7 +62,7 @@ export const Controlled = () => {
   const [value, setValue] = React.useState(['tag1', 'tag2'])
 
   return (
-    <div className="flex items-center space-x-8">
+    <div className="flex items-center purple-x-8">
```

Comment:
Aqui não é cor! É a regra de espaçamento do tailwind 😅 
```suggestion
    <div className="flex items-center space-x-8">
```
https://tailwindcss.com/docs/space

Reply:
Inclusive essa mudança se repetiu em outros lugares aqui no PR